src/platform/intel/cavs/include/cavs/lib/memory.h

```diff
@@ -45,10 +45,17 @@
 #define PLATFORM_HPSRAM_SEGMENTS	((PLATFORM_HPSRAM_EBB_COUNT \
 	+ EBB_BANKS_IN_SEGMENT - 1) / EBB_BANKS_IN_SEGMENT)
```

Comment:
Please fix commit message:

1) Add proper commmit tag e.g "lib: 
2) Use imperative mood (https://chris.beams.io/posts/git-commit/#imperative)
3) Commit message is too long

So, commit message should be something like this

lib: Make sure shifted quantity is treated as 64bit type

Append type to constants in order to make sure they are treated as 64 bit integers.

Reply:
done